week-6/learning-outcomes.md

```diff
@@ -4,12 +4,13 @@
 
 ### Password encryption
 
+- [ ] **We can store passwords securely**
 - [ ] We can hash and salt passwords using bcrypt
 - [ ] We describe why it's important to hash and salt passwords
```

Comment:
Not sure how to test for this. How about, *"We can demonstrate how to execute a successful attack on an unencrypted password field"*? (or is that a stretch goal?)

Reply:
Hmm that might be a bit much for them at this point. That would probably come under "describe why hashing is important".

The "top-level" bolded goals are all slightly more generic as they're meant to represent the overarching theme for that section. This perhaps makes them a bit less testable, so maybe they should instead be used as headings or something?